src/BLSSignatureChecker.sol

```diff
@@ -74,8 +78,8 @@ contract BLSSignatureChecker is IBLSSignatureChecker {
         BN254.G1Point memory apk = BN254.G1Point(0, 0);
         for (uint i = 0; i < quorumNumbers.length; i++) {
             require(
-                stakeRegistry.quorumUpdateTimestamp(uint8(quorumNumbers[i])) + 1 weeks <= block.timestamp,
-                "BLSSignatureChecker.checkSignatures: StakeRegistry updates must be within a week"
+                registryCoordinator.quorumUpdateTimestamp(uint8(quorumNumbers[i])) + delegation.withdrawalDelayBlocks() <= block.timestamp,
```

Comment:
this needs to be a block, not a timestamp to add blocks to it

Reply:
fixed https://github.com/Layr-Labs/eigenlayer-middleware/pull/62/commits/7db7e0a73720d6cf598a43e8eae52399ecf138fb although doesn't compile at the moment while lib/eigenlayer-contracts is pointed to master branch instead of m2-mainnet. I added the interface function here https://github.com/Layr-Labs/eigenlayer-contracts/pull/344